examples/CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ blt_add_executable(
   SOURCES aligned_allocator.cpp
   DEPENDS_ON ${example_depends})
 
+blt_add_executable(
```

Comment:
Is this required?

Reply:
No, this was left in by mistake.  I will remove it.

Reply:
Done.  Please mark resolved if you agree.